server/src/main/xsd/gateway-config-201616.xsd

```diff
@@ -1109,6 +1106,75 @@
         </complexContent>
     </complexType>
 
+    <complexType name="turn.proxyServiceType">
+        <annotation>
+            <documentation>This service implements a proxy for STUN/TURN protcols.</documentation>
+        </annotation>
+
+        <complexContent>
+            <restriction base="gateway:ServiceType">
+                <sequence>
+                    <element maxOccurs="1" minOccurs="1"
+                             name="name"
+                             type="gateway:CollapsedString"/>
+
+                    <element maxOccurs="1" minOccurs="0"
+                             name="description"
+                             type="gateway:CollapsedString"/>
+
+                    <element maxOccurs="unbounded" minOccurs="1"
+                             name="accept"
+                             type="anyURI"/>
+
+                    <element maxOccurs="unbounded" minOccurs="1"
+                             name="connect"
+                             type="anyURI"/>
+
+                    <!-- TODO is this supported ? -->
+                    <element maxOccurs="unbounded" minOccurs="0"
+                             name="balance"
+                             type="anyURI"/>
+
+                    <!-- TODO is this supported ? -->
+                    <element maxOccurs="unbounded" minOccurs="0"
+                             name="notify"
+                             type="anyURI"/>
+
+                    <element fixed="turn.proxy" maxOccurs="1" minOccurs="1"
+                             name="type"
+                             type="gateway:CollapsedString"/>
+
+                    <element maxOccurs="1" minOccurs="0"
+                             name="properties"
+                             type="gateway:turn.proxyServicePropertiesType"/>
+
+                    <element maxOccurs="1" minOccurs="0"
+                             name="accept-options"
+                             type="gateway:ServiceAcceptOptionsType" />
+
+                    <element maxOccurs="1" minOccurs="0"
+                             name="connect-options"
+                             type="gateway:ServiceConnectOptionsType" />
+
+                    <!-- TODO is this supported ? -->
+                    <element maxOccurs="1" minOccurs="0"
+                             name="realm-name"
+                             type="gateway:CollapsedString" />
+
+                    <!-- TODO is this supported ? -->
```

Comment:
realm, balance, and authorization-constraint might come into play in the future.  cross-site and notify are not, but I think they might have to be in the current form of the xsd otherwise it won't compile...?  (perhaps, put a todo to remove them, see https://github.com/kaazing/roadmap/issues/927 for details)